data/azure/instance_types.json

```diff
@@ -97,18 +120,126 @@
+  "Standard_D2d_v4": {"up": "Standard_D4d_v4", "down": null, "superseded": null},
+  "Standard_D4d_v4": {"up": "Standard_D8d_v4", "down": "Standard_D2d_v4", "superseded": null},
+  "Standard_D8d_v4": {"up": "Standard_D16d_v4", "down": "Standard_D4d_v4", "superseded": null},
+  "Standard_D16d_v4": {"up": "Standard_D32d_v4", "down": "Standard_D8d_v4", "superseded": null},
+  "Standard_D32d_v4": {"up": "Standard_D48d_v4", "down": "Standard_D16d_v4", "superseded": null},
+  "Standard_D48d_v4": {"up": "Standard_D64d_v4", "down": "Standard_D32d_v4", "superseded": null},
+  "Standard_D64d_v4": {"up": null, "down": "Standard_D48d_v4", "superseded": null},
+  "Standard_D2ds_v4": {"up": "Standard_D4ds_v4", "down": null, "superseded": null},
+  "Standard_D4ds_v4": {"up": "Standard_D8ds_v4", "down": "Standard_D2ds_v4", "superseded": null},
+  "Standard_D8ds_v4": {"up": "Standard_D16ds_v4", "down": "Standard_D4ds_v4", "superseded": null},
+  "Standard_D16ds_v4": {"up": "Standard_D32ds_v4", "down": "Standard_D8ds_v4", "superseded": null},
+  "Standard_D32ds_v4": {"up": "Standard_D48ds_v4", "down": "Standard_D16ds_v4", "superseded": null},
+  "Standard_D48ds_v4": {"up": "Standard_D64ds_v4", "down": "Standard_D32ds_v4", "superseded": null},
+  "Standard_D64ds_v4": {"up": null, "down": "Standard_D48ds_v4", "superseded": null},
+  "Standard_E2_v4": {"up": "Standard_E4_v4", "down": null, "superseded": null},
+  "Standard_E4_v4": {"up": "Standard_E8_v4", "down": "Standard_E2_v4", "superseded": null},
+  "Standard_E8_v4": {"up": "Standard_E16_v4", "down": "Standard_E4_v4", "superseded": null},
+  "Standard_E16_v4": {"up": "Standard_E20_v4", "down": "Standard_E8_v4", "superseded": null},
+  "Standard_E20_v4": {"up": "Standard_E32_v4", "down": "Standard_E16_v4", "superseded": null},
+  "Standard_E32_v4": {"up": "Standard_E48_v4", "down": "Standard_E20_v4", "superseded": null},
+  "Standard_E48_v4": {"up": "Standard_E64_v4", "down": "Standard_E32_v4", "superseded": null},
+  "Standard_E64_v4": {"up": null, "down": "Standard_E48_v4", "superseded": null},
   "Standard_E2_v3": {"up": "Standard_E4_v3", "down": null, "superseded": null},
   "Standard_E4_v3": {"up": "Standard_E8_v3", "down": "Standard_E2_v3", "superseded": null},
   "Standard_E8_v3": {"up": "Standard_E16_v3", "down": "Standard_E4_v3", "superseded": null},
-  "Standard_E16_v3": {"up": "Standard_E32_v3", "down": "Standard_E8_v3", "superseded": null},
-  "Standard_E32_v3": {"up": "Standard_E64_v3", "down": "Standard_E16_v3", "superseded": null},
-  "Standard_E64_v3": {"up": null, "down": "Standard_E32_v3", "superseded": null},
+  "Standard_E16_v3": {"up": "Standard_E20_v3", "down": "Standard_E8_v3", "superseded": null},
+  "Standard_E20_v3": {"up": "Standard_E32_v3", "down": "Standard_E8_v3", "superseded": null},
+  "Standard_E32_v3": {"up": "Standard_E48_v3", "down": "Standard_E20_v3", "superseded": null},
+  "Standard_E48_v3": {"up": "Standard_E64_v3", "down": "Standard_E20_v3", "superseded": null},
```

Comment:
The value for "down" should be "Standard_E32_v3" instead of "Standard_E20_v3"

Reply:
Done